go/consensus/tendermint/tendermint.go

```diff
@@ -33,21 +34,29 @@ import (
 	"github.com/oasislabs/oasis-core/go/common/node"
 	"github.com/oasislabs/oasis-core/go/common/pubsub"
 	cmservice "github.com/oasislabs/oasis-core/go/common/service"
+	"github.com/oasislabs/oasis-core/go/consensus/tendermint/abci"
+	"github.com/oasislabs/oasis-core/go/consensus/tendermint/api"
+	"github.com/oasislabs/oasis-core/go/consensus/tendermint/beacon"
+	"github.com/oasislabs/oasis-core/go/consensus/tendermint/crypto"
+	"github.com/oasislabs/oasis-core/go/consensus/tendermint/db"
+	"github.com/oasislabs/oasis-core/go/consensus/tendermint/epochtime"
+	"github.com/oasislabs/oasis-core/go/consensus/tendermint/epochtime_mock"
+	"github.com/oasislabs/oasis-core/go/consensus/tendermint/keymanager"
+	tmregistry "github.com/oasislabs/oasis-core/go/consensus/tendermint/registry"
+	tmroothash "github.com/oasislabs/oasis-core/go/consensus/tendermint/roothash"
+	"github.com/oasislabs/oasis-core/go/consensus/tendermint/scheduler"
+	"github.com/oasislabs/oasis-core/go/consensus/tendermint/service"
+	"github.com/oasislabs/oasis-core/go/consensus/tendermint/staking"
 	epochtimeAPI "github.com/oasislabs/oasis-core/go/epochtime/api"
 	genesis "github.com/oasislabs/oasis-core/go/genesis/api"
 	"github.com/oasislabs/oasis-core/go/genesis/file"
-	keymanager "github.com/oasislabs/oasis-core/go/keymanager/api"
-	registry "github.com/oasislabs/oasis-core/go/registry/api"
-	roothash "github.com/oasislabs/oasis-core/go/roothash/api"
-	scheduler "github.com/oasislabs/oasis-core/go/scheduler/api"
-	staking "github.com/oasislabs/oasis-core/go/staking/api"
-	"github.com/oasislabs/oasis-core/go/tendermint/abci"
-	"github.com/oasislabs/oasis-core/go/tendermint/api"
-	"github.com/oasislabs/oasis-core/go/tendermint/crypto"
-	"github.com/oasislabs/oasis-core/go/tendermint/db"
-	"github.com/oasislabs/oasis-core/go/tendermint/epochtime"
-	"github.com/oasislabs/oasis-core/go/tendermint/epochtime_mock"
-	"github.com/oasislabs/oasis-core/go/tendermint/service"
+	keymanagerAPI "github.com/oasislabs/oasis-core/go/keymanager/api"
+	"github.com/oasislabs/oasis-core/go/registry"
```

Comment:
So some are named `<backend>API` and then the tendermint version is imported as `<backend>` but some are named `<backend>` and then the tendermint version is imported as `tm<backend>`. I think it should be consistent.

Reply:
I changed to: api packages are all `<backend>API`, tendermint backends are all `tm<backend>`